plugin/zoom/lib/zoom_plugin.class.php

```diff
@@ -0,0 +1,572 @@
+<?php
+
+/* For licensing terms, see /license.txt */
+
+use Chamilo\PluginBundle\Zoom\API\CreatedRegistration;
+use Chamilo\PluginBundle\Zoom\API\JWTClient;
+use Chamilo\PluginBundle\Zoom\API\MeetingSettings;
+use Chamilo\PluginBundle\Zoom\API\ParticipantList;
+use Chamilo\PluginBundle\Zoom\API\ParticipantListItem;
+use Chamilo\PluginBundle\Zoom\API\PastMeeting;
+use Chamilo\PluginBundle\Zoom\CourseMeeting;
+use Chamilo\PluginBundle\Zoom\CourseMeetingInfoGet;
+use Chamilo\PluginBundle\Zoom\CourseMeetingList;
+use Chamilo\PluginBundle\Zoom\CourseMeetingListItem;
+use Chamilo\PluginBundle\Zoom\File;
+use Chamilo\PluginBundle\Zoom\Recording;
+use Chamilo\PluginBundle\Zoom\RecordingList;
+use Chamilo\PluginBundle\Zoom\UserMeetingRegistrant;
+use Chamilo\PluginBundle\Zoom\UserMeetingRegistrantListItem;
+
+class ZoomPlugin extends Plugin
```

Comment:
Codacy found an issue: [The class ZoomPlugin has a coupling between objects value of 15. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)